spec/support/helpers.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Helpers
+  def fill_in_sign_up_form(user)
```

Comment:
```suggestion
  def fill_in_sign_up_form_as(user)
```
Czy tak byłoby bardziej czytelnie?

Reply:
imo tak